app/common/directives/ush-logo-react/ush-logo.jsx

```diff
@@ -1,16 +1,36 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+//import { connect } from "react-redux";
+import { bindActionCreators } from 'redux';
+import Uirouter from '@uirouter/angularjs';
+
+import connectWithStore from '../../../react-transition/connectWithStore.jsx';
+import * as UshLogoActions from './ush-logo.actions';
+import { getFormsFromState } from './ush-logo.reducer';
 
 const MyComponent = props => (
   <div>
+    {props.forms[0].id}
     <p className="foo">Foo: {props.foo}</p>
     <p className="baz">Baz: {props.baz}</p>
   </div>
 );
 
-export { MyComponent as default };
+const mapStateToProps = (state) => {
+    return {
+        forms: getFormsFromState(state)
+    }
+};
+
+function mapDispatchToProps(dispatch) {
+    return {
+        UshLogoActions: bindActionCreators(UshLogoActions, dispatch),
+    };
+};
 
 MyComponent.propTypes = {
   foo: PropTypes.number.isRequired,
   baz: PropTypes.number.isRequired
 };
+
+export default connectWithStore(MyComponent, mapStateToProps, mapDispatchToProps);
```

Comment:
I couldn't figure out if it was possible to make this work with connect style syntax ie.
`connect(mapStateToProps, mapDispatchToProps)(MyComponent)`
so for now it has the component as the first prop instead.